### PR TITLE
readme added quick demo

### PR DESCRIPTION
**Pull request by raitalharehman**
I want to merge this change because...

It will help in quick demonstration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ These instructions will get you a copy of the project up and running on your loc
 - A running instance of Saleor.
 
     To run the storefront, you have to set the `BACKEND_URL` environment to point to the Saleor instance. If you are running Saleor locally with the default settings, set `BACKEND_URL` to: `http://localhost:8000/`.
+    <br>**For Quick Demo** use `BACKEND_URL` = `https://demo.getsaleor.com` <br>
+    **Or** `https://demo.getsaleor.com/graphql` for `API_URL` in src/index.tsx
```
Comment (NyanKiyoshi) [inline README.md]: I feel like we shouldn't tell the user to change the `API_URL` constant as it's not a proper way to proceed. Talking about changing the source code is too low level for the user for this kind of documentation. This documentation should only be for high level interfaces.

Comment (maarcingebala) [reply README.md]: I agree about not mentioning `API_URL` change, could we remove that? We could mention though that people may try the storefront with the public demo as a backend, but I'd rephrase that a little. How about:

```
You can also use the public demo instance as a backend, but it runs in read-only mode and some features may be unavailable. To use it, set `BACKEND_URL` to `https://demo.getsaleor.com/`.
```

File: README.md [hunk 2]
```diff
@@ -66,7 +68,7 @@ Go to `http://localhost:3000` to access the storefront.
 
 ## License
 
-This project is licensed under the BSD-3-Clause License - see the [LICENSE.md](LICENSE.md) file for details
+This project is licensed under the BSD-3-Clause License - see the [LICENSE](LICENSE) file for details
```
Comment (maarcingebala) [inline README.md]: Could you also put a period at the end of that sentence? It is missing now.
